src/styles/tab.css

```
@@ -0,0 +1,32 @@
+@import "./colors.css";
+
+.tab {
+  display: flex;
+  margin: 32px 0;
+}
+
+.tab-item {
+  cursor: pointer;
+  user-select: none;
+
+  display: flex;
```

Stylelint (17.5.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)